src/js/issues.js

```diff
@@ -50,38 +36,66 @@ function labelsToColumns(issue) {
 // Main function iterating through the issues from GitHub API,
 // pulling out the important information 
 // and rendering each issue using the Handlebars template
-function groupIssuesByLanguage(issues) {
+function groupIssuesByTopic(issues) {
     $.each(issues, function(key, val) {
         issue = labelsToColumns(val);
 
         renderIssue(issue);    
     });
 }
 
-// Append the give issue to the appropriate Language list
+// Append the give issue to the appropriate Topic list
 function renderIssue(issue) {
     var rendered_issue = Handlebars.templates.issues(issue);
-    issue.lang = (issue.lang === "C++")? "Cplusplus" : issue.lang;
-    var $lang_group = $("#" + issue.lang);
-    $lang_group.removeClass("hide");
-    $lang_group.children("ul").append(rendered_issue);
+    var $topic_group = $(sanitizeId(issue.topic));
+    $topic_group.removeClass("hide");
+    $topic_group.children("ul").append(rendered_issue);
 }
 
-// Render the containers for each Language list
-function renderLanguages() {
-    var lang_obj = {};
-    lang_obj.languages = langs;
+// Render the containers for each Topic list
+function renderTopics(topics) {
+    var topic_obj = {};
+    topic_obj.topics = topics;
+    
+    var rendered_topics = Handlebars.templates.topic_groups(topic_obj);
+    $("#issues_list").html(rendered_topics);
+}
+
+// Generate a list of topics
+function generateTopics(issues) {
+    var re = new RegExp("Topic: (.*)"); // matches CSS in "Topic: CSS"
+    var topics = [];
+
+    $.each(issues, function (key, issue) {
+        // let's assume that we will never have more than one topic assigned to an issue
+        for (var i = issue.labels.length - 1; i >= 0; i--) {
+            var re_result = issue.labels[i].name.match(re);
+            if (re_result) {
+                var topic = re_result[1];
+                if ($.inArray(topic, topics) === -1) { // $.inArray returns index if found; -1 otherwise
+                    topics.push(topic);
+                }
+            }
+        }
+    });
+
+    topics.sort();
     
-    var rendered_langs = Handlebars.templates.lang_groups(lang_obj);
-    $("#issues_list").html(rendered_langs);
+    return topics;
+}
+
+// The following function takes care of escaping these characters and places a "#" at the beginning of the ID string
+function sanitizeId(myid) {
+    // it is possible for a topic label to not exist
+    // a topic C# becomes C\#; C-- becomes C\-\-
+    return (myid) ? "#" + myid.replace( /(:|#|\+|\.|\[|\]|-|,|=|@)/g, "\\$1" ).replace( /\s/g, "_") : "";
```

Comment:
We might want to add a final replace to remove/repalce anything remaining that's non alphanumeric to prevent any page breakage. 

If we create a topic with any other bad characters (e.g. `()%^&*{}`) it breaks the page, like we currently experience with `C#`:

<img width="1422" alt="duckduckhack_tasks" src="https://cloud.githubusercontent.com/assets/873785/23040257/b468ec9c-f45e-11e6-88cb-39ac0b6040e2.png">